packages/offix-datastore/src/ModelSchema.ts

```diff
@@ -0,0 +1,129 @@
+import invariant from "tiny-invariant";
+import { JSONSchema7 } from "json-schema";
+
+export interface DSProperties extends JSONSchema7 {
+  index?: boolean;
+  primary?: boolean;
+  default?: any;
+  encrypted?: boolean;
+  key?: string;
+}
+
+export declare class DSJsonSchema<T> {
+  name: string;
+  namespace?: string;
+  version: number;
+  indices?: string[];
+  encrypted?: string[];
+  primaryKey?: string;
+  type: "object";
```

Comment:
Create type and add scalar as well

Reply:
@wtrocki type is there already, will add scalar